tests/unit9_task.spec.js

```diff
@@ -0,0 +1,46 @@
+import {test, expect} from '@playwright/test';
+
+test.describe('Check Products title and Add Products to the cart', () => {
+
+    test.beforeEach('Login as standart_user', async ({ page }) => {
+        await page.goto('https://www.saucedemo.com/');
+        await page.getByPlaceholder('Username').fill('standard_user');
+        await page.getByPlaceholder('Password').fill('secret_sauce');
+        await page.getByTestId('login-button').click();
+    });
+
+    test('Products title is displayed', async ({ page }) => {
+        await expect(page).toHaveURL('https://www.saucedemo.com/inventory.html');
+        await expect(page.getByTestId('title')).toHaveText('Products');
+    });
+
+    test('Shopping Cart icon is displayed', async ({ page }) => {
+        await expect(page.locator('.shopping_cart_link')).toBeVisible();
+    });
+
+    test('More than 1 product is displayed', async ({ page }) => {
+        const productItemsCount = await page.getByTestId('inventory-item').count();
+        expect(productItemsCount).toBeGreaterThan(1);
+    });
+
+    test('Test 2 - Add product to the cart', async ({ page }) => {
+
+        const listOfProducts = await page.locator('[data-test="inventory-item"]').all();
```

Comment:
to avoid using `all` you can use `nth()` method 
check this - https://playwright.dev/docs/locators#get-by-nth-item

Reply:
Done